md/bpm-api.md

```diff
@@ -2814,13 +2810,18 @@ You can get a case by using its identifier. Request url
 * **Request Payload**  
   The process definition id, in JSON
 
-  Create a case without variables
+##### Create a case without variables
   ```json
   {
     "processDefinitionId":"5777042023671752656"
   }
   ```
-  Create a case with variables
+##### Create a case with variables
+
+::: danger
+:fa-exclamation-triangle: **Warning:** The attribute "variables" on the request payload is used to initialize the process variables (not BDM variables).
```

Comment:
I would add `If you want to initialize BDM variables at process instantiation, add a contract on the process and map BDM variables to the contract data. See [Start a process using an instantiation contract](#start-process-with-contract) for usage.`